content/en/news/releases/1.5.x/announcing-1.5/change-notes/index.md

```diff
@@ -4,4 +4,116 @@ description: Istio 1.5 release notes.
+- Remove endpoints/ingresses/nodes from annotations analyzer  #18930.
+- Skip Analyzers that require inputs not in the current snapshot  #18961.
+- Fix `mcpc` tools error #19032.
+- Use specific message when sidecar is missing #19428.
+- Update analyzer `IsIstioControlPlane` filter to also look for `release=istio` #19906.
+- Fixing race conditions in Analyzing Distributor tests  #20136.
+- Implement extra environment variables from `ProxyMetadata` #20447.
+- Check for err when adding policy in tests #18478.
+- Add unit test for validation analyzer wrapper #18801.
+- Require the namespace for the webhook secret #18901.
+- add test to help enforce that new analyzers are added to All() #19239.
+- Analyzer test harness supports custom mesh configuration, add test case for … #19508.
+- Don't validate `credentialName` in gateway secrets analyzer if it isn't specified  #19905.
+- Make headless service passthrough resolution optional #19992.
+- Put chunk count back into error message in `kubesource` #19358.
+- clean up logging for service port conversion  #19479.
+- Fix typo in analyzer name #19557.
+- Don't instruct users to delete the pod  #19584.
+- Update analysis `readme` to match #19729 #19760.
+- add quotation marks for PATH #19433  #19774.
+- Allow re-use of `base.yaml` #19858.
+
+
+## `area/perf and scalability`
+
+- **Improved** Skip unused services while generate clusters for the gateways #20124.
+- **Improved** Skip calling `updateEDS` for headless service #18952.
+- disable `SNI-DNAT` at ingress gateway by default #18431.
+- The declaration of err overrides #19473.
+- create slice with capacity, when capacity is known #18504.
+
+## `area/test and release` - Triage done
+
+- Build a docker image for `istioctl` #19079.
+
+## `istioctl`
+- **Added** Introduce mutual TLS analyzer #18917.
+- **Added** `ServiceAssociation` Analyzer #19383.
+- **Added** add `PortNameAnalyzer` #19375.
+- **Added** more validation rules for `RequestAuthentication` #19369.
+
+- **Added** support for multiple input files to `istioctl manifest` #20190.
+- **Replaced** the `IstioControlPlane` API with the `IstioOperator` API.
+- **Added** `istioctl analyze -L` to show a list of all analyzers available #19200.
+- **Added** the ability to suppress messages from `istioctl analyze` #19673.
+- **Improved** the `bookinfo` demo to run without root permissions #16326.
+- **Added** structured format options to `istioctl analyze` #18700.
+- Use annotation methods provided by Istio API in analyzer #18829.
+- Fix bug where analyzer suppressed cluster-level resource messages. #18935.
+- **Added** links to relevant documentation to `istioctl analyze` output #19105.
+- add selector for `istioctl` dashboard #19191.
+- **Added** a new flag `--all-namespaces` to `istioctl analyze` to analyze the entire cluster #19209.
+- **Added** support for analyzing content passed via `stdin` to `istioctl analyze` #19393.
+- **Graduated** `istioctl analyze` out of experimental  #19488.
```

Comment:
```suggestion
- **Added** mTLS analyzer - #18350
- **Added** JwtAnalyzer - #20812
- **Added** ServiceAssociationAnalyzer #19383
- **Added** SercretAnalyaer #19583
- **Added** sidecar ImageAnalyzer  #20929
- **Added** PortNameAnalyzer #19375
- **Added** Policy DeprecatedAnalyzer #20919
- **Updated** more validation rules for `RequestAuthentication` #19369.

- **Graduated** `istioctl analyze` out of experimental  #19488.

- **Added** a new flag `--all-namespaces` to `istioctl analyze` to analyze the entire cluster #19209.
- **Added** support for analyzing content passed via `stdin` to `istioctl analyze` #19393.
- **Added** `istioctl analyze -L` to show a list of all analyzers available #19200.
- **Added** the ability to suppress messages from `istioctl analyze` #19673.
- **Added** structured format options to `istioctl analyze` #18700.
- **Added** links to relevant documentation to `istioctl analyze` output #19105.
- **Updated** annotation methods provided by Istio API in analyzer #18829.
- **Fixed** bug where analyzer suppressed cluster-level resource messages. #18935.



- **Added** support for multiple input files to `istioctl manifest` #20190.
- **Replaced** the `IstioControlPlane` API with the `IstioOperator` API.
- **Added** selector for `istioctl` dashboard #19191.
```

Reply:
@elfinhe @ostromart - are operator/installer changes reported under istioctl or their own section?

Reply:
In the past it was under an installation section because the changes usually apply to both istioctl and the controller. 
I'll add a section for operator and @fpesce can decide where to put the items in the end.